packages/backend/src/services/SemanticLayerService/Pivoting.ts

```diff
@@ -44,9 +49,15 @@ export const groupResults = (
 ): SemanticLayerResultRow[] => {
     const df = pl.DataFrame(results);
 
-    const aliasedAggs: pl.Expr[] = values.map((v) =>
-        pl.col(v.name)[v.aggFunction]().alias(`${v.aggFunction}(${v.name})`),
-    );
+    const aliasedAggs: pl.Expr[] = values.map((v) => {
```

Comment:
@almeidabbm and @IrakliJani why is this still here? I think it's confusing, we don't need this aggregation and developers shouldn't need to worry about it.